### DISCUSSION TITLE
Fields that should not be changed

### DISCUSSION THREAD

**Discussion by AlexAS81** [OP-TED/eForms-SDK#687] [Q&A] [answered]
In the workshop on September 26, it was commented that the following fields should not generate a change type notice: BT-04 Procedure ID, BT-03 Form Type, BT-02 Notice Type, OPT-070 Notice Subtype, BT-105 Procedure Type, BT-23 Main Nature, BT-702 Notice language, BT-1501-(n)-Contract previous notice, BT-142 Winner chosen and the BT-195 Unpublished identifiers.

What should be generated in these cases? A notice with a new noticeId and same procedureId? New procedureId and noticeId?

For example, if BT-142 Winner chosen varies, how should I proceed? Send a new subtype 29 with the same noticeI and procedureId or with new ones?

Karl Ferrand spoke on the subject but it was not clear to me.

Thanks in advance

**Comment by YvesJo** [comment]
The Change notice is a new notice on its own and requires its own notice ID; it belongs to the same procedure as the changed notice and both notices will share the same Procedure ID.
Please, for further non-technical questions directly get in touch with the helpdesk

**Reply by AlexAS81** [reply]
The question really is what to do with the notices in which one of these fields changes and does not generate a change notice.

**Reply by YvesJo** [reply]
Then you may have to cancel the notice (i.e. make it void) and create a new one or even cancel the procedure and start a new one.

**Reply by AlexAS81** [reply]
So what would be more correct? Cancel the notice or create a new one?

In both cases would a new procedure have to be created?

Can you show us an example of both cases?

**Reply by YvesJo** [reply, marked as answer]
please get in touch with the helpdesk ted@publications.europa.eu for non-technical questions.
Thanks for your understanding.
